src/app/Http/Requests/ResendVerificationRequest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class ResendVerificationRequest extends FormRequest
+{
+    public function authorize()
+    {
+        return true;
+    }
+
+    public function rules()
+    {
+        return [];
+    }
+}
+
```

Comment:
Why this request is empty?? Do you have to resend the verification request without validating it??

Reply:
I already resolve this sir, Thank you!